en/docs/deploy/secure-userstore-using-pbkdf2.md

```diff
@@ -0,0 +1,82 @@
+# PBKDF2 hashing
+Password-Based Key Derivation Function 2 (PBKDF2) hashing algorithm is a modern hashing algorithm recommended by NIST.  We can use PBKDF2 hashing method to securely store user passwords in user stores. This method reduces the risk of brute-force attacks due to insecure passwords.
```

Comment:
Check whether we have two spaces here > `NIST.  We can`

Reply:
```suggestion
Password-Based Key Derivation Function 2 (PBKDF2) hashing algorithm is a modern hashing algorithm recommended by NIST. We can use the PBKDF2 hashing method to securely store user passwords in user stores. This method reduces the risk of brute-force attacks due to insecure passwords.
```